File: client/src/app/_dtos/admin/interviewAttendanceDto.ts
import { IntervwItemCandidate } from "src/app/_models/hr/intervwItemCandidate";

export interface IInterviewAttendanceDto
{
    intervwItemId: number;
    intervwItemCandidateId: number;
    interviewItemCandidateId: number;
    checked: boolean;
    customerName: string;
    orderNo: number;
    interviewId: number;
    orderItemId: number;
    interviewVenue: string;
    professionName: string;
    scheduledFrom: Date;
    interviewMode: string;
    reportedAt?: Date;
    interviewedAt?: Date;
    personId: string;
    applicationNo: number;
    candidateName: string;
    interviewStatus: string;
    interviewerRemarks: string;
    attachmentFileNameWithPath: string;
}

File: api/DTOs/HR/InterviewAttendanceDto.cs
using api.Entities.Admin;

namespace api.DTOs.HR
{
    public class InterviewAttendanceDto
    {
        public int OrderNo { get; set; }
        public string CustomerName { get; set; }
        public int Id { get; set; }
        public int InterviewId { get; set; }
        public string InterviewVenue { get; set; }
        public int OrderItemId { get; set; }
        //public int ProfessionId { get; set; }
        public string ProfessionName { get; set;}   
        public DateTime ScheduledFrom { get; set; }
        //public DateTime ScheduledUpto { get; set; }
        public DateTime ReportedAt { get; set; }  
        public DateTime InterviewedAt { get; set;}
        public string InterviewMode { get; set; }
        //public string InterviewerName { get; set; }
        //public int CandidateId { get; set; }
        public string PersonId { get; set; }
        public int InterviewItemCandidateId { get; set; }
        public int ApplicationNo { get; set; }
        public string CandidateName { get; set; }
        public string InterviewStatus { get; set; }
        public string InterviewerRemarks { get; set; }
        public string AttachmentFileNameWithPath { get; set; }
    }
}